fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/DimensionsServlet.java

```diff
@@ -332,6 +326,27 @@ public Response getDimensionRows(
         }
     }
 
+    /***
+     * Get the pagination of dimension rows.
```

Comment:
Description is good.

Detailed description should start in a different paragraph saying "if there is filter in API request, then... otherwise, ..."